python/src/dictionary.rs

```diff
@@ -435,9 +435,7 @@ fn config_repr(cfg: &Config) -> Result<String, std::fmt::Error> {
         Ok(dicts) => {
             for (i, dic) in dicts.iter().enumerate() {
                 write!(result, "{}", dic.display())?;
-                if i + 1 == dicts.len() {
-                    write!(result, "]")?;
-                } else {
+                if i + 1 != dicts.len() {
```

Comment:
Wouldn't `i < dict.len()` be better?

Reply:
Sorry, `i < dict.len() - 1`. Both are equally complicated, so you probably don't need to fix it.